packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/index.tsx

```diff
@@ -69,6 +68,7 @@ export default function SpanDetail(props: SpanDetailProps) {
       value: formatDuration(relativeStartTime),
     },
   ];
+  const deepLinkCopyText = `${window.location.origin}${window.location.pathname}?uiFind=${spanID}`;
```

Comment:
I guess I'm just used to seeing `window.location` instead of `location` on its own, but `location` on its own works fine and is shorter so I updated.

I agree about consolidating `uiFind` in its own PR. Should that just be when `uiFind` is in a string? If so, this seems to be the only instance based on a quick search (`ag --ts '['"'"'"`].+uiFind'`). But I think it would make sense to be a constant whenever it is interacting with the URL (i.e.: here and when interacting with `queryString.parse` / `queryString.stringify`)

Reply:
Also, eslint dislikes `location` but doesn't mind `window.location`. What is your preference on using `window.location` vs `// eslint-disable-line no-restricted-globals`